### DISCUSSION TITLE
No +86 region code for 2FA?

### DISCUSSION THREAD

**Discussion by jfioasd** [community/community#68037] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

There's no `+86` (China) region code in SMS authentication of 2FA.

**Comment by kamlendras** [comment]
Thank You,
Please if you are in china or use a Chinese mobile number ,
then install GitHub mobile app for 2 factor authentication.
Note: MARK THIS AS ANSWERED AND CLOSE THIS ISSUE ;)

**Comment by tiagoangelos** [comment]
Before trying to find a solution, read this article, I hope it helps.
https://dciber.org/por-que-voce-nao-deve-usar-sms-para-2fa/

**Comment by 743859910** [comment]
安装并登录Github App，后续有需要验证的时候Github网站上会显示两位数字的验证码，在App输入批准就可以了。
